### DISCUSSION TITLE
Avalonia WebBrowser app blank with no error

### DISCUSSION THREAD

**Discussion by duongphuhiep** [AvaloniaUI/Avalonia#16828] [Q&A]
Hello,

I'm playing around to evaluate Avalonia for my future project.

* [My desktop application](https://github.com/duongphuhiep/LearnDesktopApp/tree/master/AvaloniaApplication1.Desktop) and [Android App](https://github.com/duongphuhiep/LearnDesktopApp/tree/master/AvaloniaApplication1.Android) work well
* But the [Web application](https://github.com/duongphuhiep/LearnDesktopApp/tree/master/AvaloniaApplication1.Browser) displayed a blank page after the Splash screen "Powered by avalonia"

it was a very basic application. Because there aren't any error in the browser (Dev panel) and the Console, so I have no idea what's wrong, where to look at?

Do you have any suggestion how to diagnostic the Avalonia Web version?

**Comment by duongphuhiep** [comment]
hello 

the application magically works today, I didn't do anything... not sure what happened (chorme updated??).

expeirence learn for cross-platform development: wasm/browser is much much different from desktop/mobile platform. Meaning that

* if your app run well on Desktop => there is a good chance that it would run on Mobile, as they are similar enough.
* But the Web platform (even with wasm) is a complete "alien", Chance that your app would also run well on this platform is much slimmer! => if it works then lucky you, but you shouldn't have much expectation for it. We should bet on dedicated techno (Blazor, Nextjs, Nutxjs..) for this "alien"

Anyway, Avalonia give the best DXperience (and hopes) for cross-platform development. Things just works

**Reply by maxkatz6** [reply]
Most likely you had an issue with browser cache.
